src/components/Navbar/navbarStyles.js

```diff
@@ -11,9 +11,9 @@ export default makeStyles((theme) => ({
     homePageLink: ({ isMobile }) => ({
         marginLeft: theme.spacing(isMobile ? 0 : 3),
     }),
-    linkStyle: ({ forceDesktopLayout }) => ({
+    linkStyle: ({ isMobile }) => ({
         textDecoration: "none",
-        color: !forceDesktopLayout ? theme.palette.secondary.main : "white",
+        color: isMobile  ? theme.palette.secondary.main : "white",
```

Comment:
This `isMobile` check has problems as well as the `forceDesktopLayout`. We'll need a combination of both.

Check for example the Not Found page (by going to some random route) in desktop and mobile. Since it forces the desktop layout in mobile, the color will be wrong there. That would also happen in the homepage itself, but doesn't render a homepage button, so it is not as problematic.

I think the condition we need is `isMobile && !forceDesktopLayout`, but please review and test it in multiple pages in the different layouts